hello_world_on_chain/lib/TAParserLib.sol

```diff
@@ -20,10 +20,22 @@ library TAParserLib {
         string Output;
     }
 
+    function verifyAttestedFnCall(
```

Comment:
This interface seems, to me, to be a point of friction.  We would like to make it easy for a user to bring data on chain and so we should take in our key and then convert it, however, we should also make it easy for a user to follow the better pattern of not converting the public key to an address every time.  I am temped to have this function take in an address and make the user contract have 2 lines instead of 1 and take in an address not the bytes of the public key thoughts?